python_modules/libraries/dagster-dbt/dagster_dbt/cli/app.py

```diff
@@ -246,11 +246,11 @@ def project_scaffold_command(
             hidden=True,
         ),
     ] = False,
-    use_experimental_dbt_project: Annotated[
+    use_dbt_project: Annotated[
         bool,
         typer.Option(
             ...,
-            "--use-experimental-dbt-project",
```

Comment:
Can also just keep `--use-experimental-dbt-project` as a flag so things that use that flag won't break (e.g. extraneous docs, internal).